receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -639,3 +639,11 @@ metrics:
       value_type: int
       aggregation: cumulative
       monotonic: false
+
+  # Base
+  container.uptime:
+    enabled: true
```

Comment:
Should this be disabled by default?

Reply:
I don't know. You tell me 🙂 I figured that as a "base" metric I'd have it enabled, but I'm ok with either.

Reply:
When a new metric is added, it's better if it's marked as not enabled by default so we avoid ingesting new metrics on upgrade by mistake. We can turn it on by default in a separate PR.

Reply:
See here for more context: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#adding-a-new-default-metric-or-turning-an-existing-optional-metric-into-default

Reply:
Great. Thanks a lot for jumping in and clarifying. I disabled it. PTAL

Reply:
@atoulme would you mind taking another look please?